java/registry/src/main/java/io/opensaber/registry/app/OpenSaberApplication.java

```diff
@@ -28,7 +28,13 @@ public static void main(String[] args) {
 		context = application.run(args);
 	}
 
-	public static ApplicationContext getContext() {
+
+	/** This method return non-web application context
+	 * @return context
+	 */
+	public static ApplicationContext getAppContext() {
```

Comment:
If this is what is getting used, why need to change main method? Can we not live with what was existing earlier. Am saying we instantiate the application separately....try this if it makes sense.
main() -> getWebAppContext() => generates a web application context.
getAppContext() => generates a none application context.
Calling a main method from another function seems weird.

Reply:
Agreed, changes done. Now web-app will call main method defaulty, getAppContext have the own code which created context, now we are not calling main method from this one.